timespan/update.go

```diff
@@ -12,17 +12,31 @@ import (
 
 // UpdateTimeSpan update a time span
 func (r *ResolverForTimeSpan) UpdateTimeSpan(ctx context.Context, id int, start model.Time, end *model.Time, tags []*gqlmodel.InputTimeSpanTag, oldStart *model.Time, note string) (*gqlmodel.TimeSpan, error) {
-	timeSpan, err := timespanToInternal(auth.GetUser(ctx).ID, start, end, tags, note)
-	if err != nil {
-		return nil, err
+	// Setting the seconds to 0 when a change comes in, to not have 'wrong' Calculations
```

Comment:
I think this should be fixed in the frontend, users should be able to use the api to update with second/millisecond precision.